termsaver/__init__.py

```diff
@@ -131,6 +132,22 @@ def entryPoint():
 
 def getScreen():
     verbose = False
+    use_random = False
+    available_random_screens = [
```

Comment:
is there a reason to hard code the screen as opposed to use the get_available_screens? (this one checks the file system instead so you would not have to do this manually)

Reply:
The original intent was, so we could limit the random screens to omit any potential NSFW screens. Perhaps we could move that into a flag for the screens themselves instead?

Reply:
yep, that looks better.